oc_mirror.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# Utility functions to create a local mirror registry using 'mirror-registry' and mirror
+# a release using 'oc mirror'
+
+function add_auth_to_pull_secret() {
```

Comment:
On a second pass, I guess that the pull-secret prepared by dev-script in the WORKING_DIR should already be fine (it already contains the registry), so it could be copied directly as ./docker/config.json file

Reply:
I'd have to test that but I think it was the pull secret in $WORKING_DIR was missing
```
    "virthost.ostest.test.metalkube.org:8443": {
      "auth": "<auth value>"
    },
```